metis-common/metis-common-utils/src/main/java/eu/europeana/metis/utils/CommonStringValues.java

```diff
@@ -24,8 +24,19 @@ public final class CommonStringValues {
 
   public static final String REPLACEABLE_CRLF_CHARACTERS_REGEX = "[\r\n\t]";
 
-  public static final Pattern CRLF_PATTERN = Pattern.compile(CommonStringValues.REPLACEABLE_CRLF_CHARACTERS_REGEX);
+  public static final Pattern CRLF_PATTERN = Pattern.compile(REPLACEABLE_CRLF_CHARACTERS_REGEX);
 
   private CommonStringValues() {
   }
+
+  /**
+   * Sanitized input value from Logging injection attacks(javasecurity:S5145).
+   * <p>Replaces CR and LF characters with a safe value e.g. '_'.</p>
+   *
+   * @param input the input
+   * @return the sanitized input, safe for logging
+   */
+  public static String sanitizeCRLF(String input) {
```

Comment:
do we want `american english` or `british english` :smile: ? `sanitizeCRLF` or `sanitiseCRLF`, also in the comment `sanitised`

Reply:
:D For this specific difference I always use 'z', it makes more sense to me and in code we usually use US local. In fact almost everywhere we use 'z', although it tends to be 's' in some documents.